art/psa-boot.ascii-art

```diff
@@ -0,0 +1,13 @@
+       TE_i       MBL           MBS
+         |         |             |
+.--------|---------|-------------|----.
+| loop i |         |             |    |
+|        | measure |             |    |
+|        | & load  |             |    |
+|        |o--------+             |    |
+|        |         | write       |    |
+|        |         | measurement |    |
+|        |         +------------>|    |
```

Comment:
```suggestion
|        |         +W----------->|    |
```

Reply:
Just for consistency I am wondering whether we should add this "W"

Reply:
In this case, there was no need to qualify the arrow as the description already mentions "write [...]". Similarly, in the other diagram, it says "read [...]".